### DISCUSSION TITLE
ABP Tenant Identity

### DISCUSSION THREAD

**Discussion by imowaisafsar** [abpframework/abp#14339] [Q&A] [answered]
I am using ABP and when I register a new tenant using pre-define APIs, a new tenant is created and a new role by the name Admin is created even If I already had an admin role in the table.

**Comment by maliming** [comment, marked as answer]
hi

The name of the role is the same but they have different `TenantId`.

https://docs.abp.io/en/abp/latest/Data-Filtering#imultitenant

**Reply by imowaisafsar** [reply]
That's right, but why does ABP create a new role for each new tenant? ABP should use the existing role id to assign user roles if there is already an admin role in DB.

**Reply by maliming** [reply]
Tenants may use different databases.
